src/screens/ImportSeedPhraseSheet.js

```diff
@@ -172,33 +203,25 @@ export default function ImportSeedPhraseSheet() {
       if (ens && ens !== input) {
         name = ens;
       }
+      showWalletProfileModal(name);
     } else {
       try {
-        const { wallet } = getWallet(input);
-        const ens = await web3Provider.lookupAddress(wallet?.address);
-        if (ens && ens !== input) {
-          name = ens;
-        }
+        setBusy(true);
+        setTimeout(async () => {
```

Comment:
FYI: had to put this in here because the getWallet fn blocks the JS thread so there is no re-render with the spinner until  after it's done, so it defeats the purpose of the spinner itself.

Tried using InteractionManager instead but didn't work  and the other approach would be to use `useEffect(() => { ..., [busy]}` but it breaks the readability of the logic for this code.